packages/runner/README.md

```diff
@@ -0,0 +1,68 @@
+# @secbox/runner
+
+Top level @secbox module for usage in test frameworks.
```

Comment:
Let's remove this line, it doesn't make any sense for the end-user

Reply:
changed also in `package.json`